src/components/layout/LayoutSlots.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { PropsWithChildren } from 'react';
+import { cssVar } from '~utils/design-tokens';
+
+export enum GlobalGridArea {
+  header = 'header',
+  banner = 'banner',
+  sidebar = 'sidebar',
+  content = 'content',
+}
+
+export enum ContentGridArea {
+  header = 'header',
+  aside = 'aside',
+  page = 'page',
+}
+
+export enum PageGridArea {
+  header = 'header',
+  main = 'main',
+  footer = 'footer',
+}
+
+export enum AsideSize {
+  small = 'small',
+  medium = 'medium',
+  large = 'large',
+}
+
+const AsideWidth = {
+  [AsideSize.small]: cssVar('layout-aside-width-small'),
+  [AsideSize.medium]: cssVar('layout-aside-width-medium'),
+  [AsideSize.large]: cssVar('layout-aside-width-large'),
+};
+
+/*
+ * Layer 1 Grid components
+ */
+export const BannerContainer = styled.div`
+  grid-area: ${GlobalGridArea.banner};
+`;
+BannerContainer.displayName = 'BannerContainer';
+
+export const Header = styled.div`
+  grid-area: ${GlobalGridArea.header};
+`;
+Header.displayName = 'Header';
+
+export const ContentWrapper = styled.div<{ fixed?: boolean }>`
+  position: relative;
+  grid-area: ${GlobalGridArea.content};
+  overflow-y: hidden;
+
+  ${(props) =>
+    props.fixed
+      ? css`
+          margin-left: auto;
+          margin-right: auto;
+          max-width: 1550px;
+        `
+      : ''}
```

Comment:
I guess the idea is to finish this implem with the multiple sizes with ticket ECHOES-920